tuner/tuner/libtuner.py

```diff
@@ -1284,9 +1283,9 @@ def parse_model_benchmark_results(
     ]
 
     dump_list = []
-    incomplete_list: list[
-        tuple[int, Optional[str]]
-    ] = []  # format: [(candidate_id, device_id)]
+    incomplete_list: list[tuple[int, Optional[str]]] = (
+        []
+    )  # format: [(candidate_id, device_id)]
```

Comment:
this should fit on a single line if you move the comment to its own line

Reply:
My pre-commit was broken locally yesterday, and I then I was having trouble getting the right version to work, so it incorrectly formatted this. I think I fixed it now, and this change is gone.